salt/beacons/diskusage.py

```diff
@@ -89,10 +89,10 @@ def beacon(config):
         mount = mounts.keys()[0]
 
         try:
-            _current_usage = psutil.disk_usage(mount)
+            _current_usage = psutil.disk_usage(_mount)
```

Comment:
I don't understand this at all. You've set this to an undefined variable.

Reply:
in the proposed fix, the mount a couple lines above this was changed to `_mount` so it seems like that was not copied to the PR.